### PR TITLE
Updated getting-started doc; included make-tom script

### PR DESCRIPTION
**Pull request by rachel3834**
Added information on the make-tom script and generally updated the getting-started page.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -48,7 +48,8 @@
 extensions = [
     'sphinx.ext.autodoc',
     'sphinx.ext.autosummary',
-    'sphinx.ext.intersphinx'
+    'sphinx.ext.intersphinx',
+    'sphinx_panels'
```
Comment (jchate6) [inline docs/conf.py:52]: This is required for link buttons.
It needs to be added to `pyproject.toml`

File: docs/introduction/getting_started.rst [hunk 2]
```diff
@@ -15,24 +15,50 @@ likely won’t need to utilize any advanced features.
...
+
+::
+
+  ./make-tom.sh my_tom
+
+You can now fast-forward to the section below on :ref:`Running in dev server <runserver>`.
 
 Installing the TOM Toolkit and Django
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If you prefer to build your own system, here's how.
```
Comment (jchate6) [inline docs/introduction/getting_started.rst:47]: maybe reword to "build your system manually". 
This kinda implies that the results of make-tom aren't yours or aren't customizable.

File: docs/introduction/getting_started.rst [hunk 3]
```diff
@@ -96,21 +126,29 @@ your current project. Run it:
...
    ./manage.py migrate
 
-Now, start the dev server:
+.. _runserver:
+
+Running the dev server
+~~~~~~~~~~~~~~~~~~~~~~
+
+Now that your TOM has been built, you’re ready to try it out!
+You can run it first of all directly on your local machine using the
```
Comment (jchate6) [inline docs/introduction/getting_started.rst:145]: maybe change this to "You can run it immediately, directly on your local..."

**Review by jchate6 — CHANGES_REQUESTED** [review]
I've taken care of the dependencies, and made a few suggestions.

File: docs/introduction/getting_started.rst (hunk 2 above)
Comment (rachel3834) [reply docs/introduction/getting_started.rst:47]: OK fair enough, I've changed the wording

File: docs/introduction/getting_started.rst (hunk 3 above)
Comment (rachel3834) [reply docs/introduction/getting_started.rst:145]: Updated wording

**Review by jchate6 — APPROVED** [review]
excellent
